### DISCUSSION TITLE
Autenticação firebase no google flutter

### DISCUSSION THREAD

**Discussion by matheusvarela** [balta-io/forum#144] [Tira Dúvidas] [answered]
Boa noite

Fiz o curso de flutter com firebase autenticando no google

Queria saber o que armazeno para identificar o usuário do app com esse tipo de autenticação, o email?

Após o usuário logar com serviço de terceiro, considero ele logado no meu app identificando pelo email?

Pensando na parte de criação de usuário no app, o que armazeno para identificar o usuário?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @matheusvarela, como vai?

Você pode armazenar o E-mail ou ID gerado para o usuário pelo Firebase. Ambos funcionam e aí depende do que você quer utilizar para reconhecer seu usuário dentro da sua API.
